### ISSUE TITLE
Delete experiment doesn't work

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#240]
In GitLab by @pablohe on Feb 16, 2017, 16:09

## I followed this steps to reproduce the bug

```
$> autosubmit expid  -H marenostrum3 -y t00o -d"test delete fail"
The new experiment "a0gt" has been registered.
Copying previous experiment config directories
Experiment registered successfully
Remember to MODIFY the config files!
$> autosubmit delete a0gt
Do you want to delete a0gt ? [y/n]
y
Removing experiment directory...
Deleting experiment from database...
Experiment a0gt deleted

> autosubmit delete a0gt
[ERROR] The experiment does not exist

```
## The observed behaviour 
When an experiment is deleted the directory is removed but you can found it  [here](http://bscesautosubmit01:8000/autosubmit_v3/default/index )

## The expected behaviour is, 
The experiment  should does not appear [here](http://bscesautosubmit01:8000/autosubmit_v3/default/index )

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 17, 2017, 15:51

Mentioned in commit fac00ac1b51c30a408b2689ec469613eb2afabee

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 17, 2017, 15:54

Hi, 

This bug was introduced by myself here: af7e41bf3459684f7d51d28faef5bc18c36c9d02, sorry!

Now it's already fixed! It will be available in the next release.

If you need to ```delete``` an experiment you can move to an Autosubmit version older than this commit, delete the experiment and move again to the current version.

Thanks!

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:18

Mentioned in commit 1a577c49078215eeb56193477cb68567d09369d6

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:29

Mentioned in commit 07d1cf7fc1bc92a0153bc7e09bcb7fc5eb6f25df

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 18:16

Fixed! 

Available since Autosubmit v3.7.8.
